Automattic/wp-calypso
# Tailwind: Add support to calypso

## Proposed Changes

This PR is a POC to add TailwindCSS support to Calypso. Tests need to be made to ensure that everything is working as expected. 

## Testing Instructions

### Test 1: General test that 
1. Start local environment
2. Add the following HTML code somewhere in the `hosting-config` ( at any component ):
```
   <h1 className="text-3xl font-bold underline">Hello world!</h1>
```
3. Ensure that you see something like below:
![tailwind-calypso](https://github.com/Automattic/wp-calypso/assets/497103/9251e1ce-89b4-4b04-92ca-e84456b0af79)

**NOTE:** It seems that `yarn.lock` makes the unit tests to fail. I'll fix that, as soon as possible


### Test 2: Cache card refactor to use tailwind
1. Start local environment
2. Visit `/hosting-config` page
3. Ensure that Cache card look the same as before, and you see the new tailwind classes as shown:
![cache_card_tailwind](https://github.com/Automattic/wp-calypso/assets/497103/68bebc70-c2b7-4edb-b244-99510930ca4f)


## Pre-merge Checklist
